pkg/operator/ceph/cluster/watcher.go

```diff
@@ -330,6 +399,80 @@ func (c *clientCluster) fenceRbdImage(
 	return nil
 }
 
+func (c *clientCluster) fenceCephFSVolume(
+	ctx context.Context, node *corev1.Node, cluster *cephv1.CephCluster,
+	clusterInfo *cephclient.ClusterInfo, cephFSPV corev1.PersistentVolume) error {
+
+	logger.Infof("cephfs PV NAME %v", cephFSPV.Spec.CSI.VolumeAttributes)
```

Comment:
How about this logging?
```suggestion
	logger.Infof("fencing cephfs volume %q on node %q", cephFSPV.Name, node.Name)
```

Reply:
Thanks Travis. Updated